src/main/java/com/matt/forgehax/mods/ActiveModList.java

```diff
@@ -82,7 +91,7 @@ public void onRenderScreen(RenderGameOverlayEvent.Text event) {
       ForgeHaxService.INSTANCE.drawWatermark(getPosX(watermarkOffsetX), getPosY(watermarkOffsetY), align);
     }
 
-    if (MC.currentScreen instanceof GuiChat || MC.gameSettings.showDebugInfo) {
+    if (condense.get() && MC.currentScreen instanceof GuiChat || MC.gameSettings.showDebugInfo) {
```

Comment:
brackets `condense.get() && (MC.currentScreen instanceof GuiChat || MC.gameSettings.showDebugInfo)`

Reply:
I think this is already fixed, please reopen if I didn't catch what you meant!

Reply:
Ye I did fix it in a newer commit